Objects/abstract.c

```diff
@@ -965,8 +965,9 @@ ternary_op(PyObject *v,
     if (z == Py_None)
         PyErr_Format(
             PyExc_TypeError,
-            "unsupported operand type(s) for ** or pow(): "
+            "unsupported operand type(s) for %.100s: "
```

Comment:
Thanks for fixing this! Really weird that `op_name` went unused so long. Would you mind adding a test for this like:

```python
        x = None
        with self.assertRaises(TypeError) as cm:
            x **= 2
        self.assertIn('unsupported operand type(s) for **=', str(cm.exception))
```

Reply:
Thanks, added a new test